CONTRIBUTING.md

```diff
@@ -106,6 +106,32 @@ python manage.py runserver
 ## Register initial user
 Once CiviWiki is running, visit the front page (probably something like http://localhost:8000). Once there, click 'log in/register', and then 'register new user'.
 
+# Creating Issues
```

Comment:
It seems to me that this section should be near the top of the contributing guide. Basically, the guide should be organized from more general to more specific. I.e. it is more likely for people to file issues before setting up a development environment.